mage_ai/frontend/components/FileTree/index.tsx

```diff
@@ -101,12 +108,13 @@ function FileTree({
   const scrollToBlock = (path: string[]) => {
     const blockPath = path.slice(1).join('/');
     const blockEl = blockRefs.current[blockPath];
-    blockEl?.current?.scrollIntoView({ behavior: 'smooth' });
+    blockEl?.current.scrollIntoView({ behavior: 'smooth' });
   };
 
   const fileTreeHandler = (path, isFolder) => (e) => {
     e.preventDefault();
     scrollToBlock(path);
+    setSelectedBlock(findBlockByPath(blocks, path));
```

Comment:
not sure if there'd ever be an undefined type error here, could possibly use optional chaining

Reply:
I had this issue earlier, it doesn't cause an error. it just makes it so that no block is selected; since this case should never arise and doesn't crash the frontend, I think it's fine (the alternative is not selecting any block anyways, so I don't think it's an issue)